package.json

```diff
@@ -13,6 +13,7 @@
     "test:watch": "turbo run test:watch"
   },
   "dependencies": {
+    "aws-amplify": "^5.0.12",
```

Comment:
Why move the `aws-amplify` out of the `apps/client/package.json`? Most packages don't use it.